play-zipkin-tracing/core/src/main/scala/jp/co/bizreach/trace/ZipkinTraceServiceLike.scala

```diff
@@ -57,14 +57,16 @@ trait ZipkinTraceServiceLike {
 
     Try(f) match {
       case Failure(t) =>
+        val timestamp = tracer.clock().currentTimeMicroseconds()
         Future {
           childSpan.tag("failed", s"Finished with exception: ${t.getMessage}")
-          childSpan.finish()
+          childSpan.finish(timestamp)
```

Comment:
finish() is usually safe operation with very low overhed, so is this an extra safeguard?